<launch>

  <arg name="mav_name" default="tupac" />
  <arg name="camera_calib_file" value="$(find rpg_calib)/cameras/bluefox_$(env CAMERA_SERIAL_NR)_pinhole.yaml"/>

  <!-- ar pose cpp node -->
  <group ns="$(arg mav_name)">

    <node name="ar_pose" pkg="ar_pose" type="ar_single" respawn="false" output="screen">
      <rosparam file="$(arg camera_calib_file)" />
      <param name="marker_pattern" type="string" value="data/4x4/4x4_55.patt"/>
      <param name="marker_width" type="double" value="135"/>
      <param name="marker_center_x" type="double" value="0.0"/>
      <param name="marker_center_y" type="double" value="0.0"/>
      <param name="threshold" type="int" value="100"/>
      <param name="use_history" type="bool" value="true"/>
      <param name="reverse_transform" type="bool" value="false"/>
      <param name="use_camera_info" type="bool" value="false"/>
      <param name="publish_visual_markers" type="bool" value="false"/>
      <param name="publish_tf" type="bool" value="true"/>
    </node>

  </group>


</launch>
